README.md

```diff
@@ -36,6 +36,13 @@ ANNOUNCEMENT_ID=[insert id here & remove brackets]
 VOICE_PARENT_ID=[insert id here & remove brackets]
 ```
 
+## Server setup
+If you are testing the bot using your own bot and not the SSE test channel, 
+you will need to create a role named "Mentor" and a role named "Online Mentor". 
```

Comment:
There's also the social role now, don't forget that guy in order to make the !social command work